src/app/helptext/network/interfaces/interfaces-form.ts

```diff
@@ -20,7 +24,9 @@ int_description_placeholder : T('Description'),
 int_description_tooltip : T('Enter a description of the interface.'),
 
 int_name_placeholder : T('Name'),
-int_name_tooltip : T('Enter a name to use for the the interface. Use the format laggX, vlanX, or bridgeX where X is a number representing a non-parent interface.'),
+int_name_tooltip : T('Read-only when editing an interface. Enter a\
```

Comment:
Again, description of what it is first, then other details like when it is read-only.